plugins/labhub.py

```diff
@@ -213,6 +213,14 @@ def create_issue_cmd(self, msg, match):
 
         if repo_name in self.REPOS:
             repo = self.REPOS[repo_name]
+            query = iss_title + " is:issue"
```

Comment:
How does this query help?  You need to search for `open` issues of `coala` repositories.

Reply:
Also, I couldn't find any implementation for GitLab. Search for appropriate method in IGitt API before you do anything.

Reply:
@Vamshi99 Yeah still working on it.Just wanted you to review the approach.Thanks

Reply:
> How does this query help? You need to search for `open` issues of `coala` repositories.

@Vamshi99 this would help us get all the issues that have been openened that contain the current issue title open or close.Because it could be that the issue we are creating has already been solved, morevoer it is no possible to include repo in the query so this would list out all issues with out title.